plugins/ftdidmx/FtdiDmxDevice.cpp

```diff
@@ -56,15 +56,19 @@ FtdiDmxDevice::~FtdiDmxDevice() {
 bool FtdiDmxDevice::StartHook() {
   unsigned int interface_count = m_widget->GetInterfaceCount();
   unsigned int successfully_added = 0;
+  unsigned int serial = 0;
 
   OLA_INFO << "Widget " << m_widget->Name() << " has " << interface_count
            << " interfaces.";
 
   for (unsigned int i = 1; i <= interface_count; i++) {
+    if (!StringToInt(m_widget->Serial(), &serial, false, 36)) {
+      OLA_WARN << "StringToInt returned false, serial used: " << serial;
```

Comment:
Don't you want to show the serial passed in, rather than the result which should be the untouched zero?

Reply:
Well I changed the function that it *does* change the output regardless of it's error state, I figured that those who care will be checking if(StringToInt) and not if(output !=0) (which would we immensely stupid). But I'll grep through the code to make sure.

Reply:
As far as I can tell all usages of StringToInt are inside if() statements.